vendor/modules.txt

```diff
@@ -198,7 +200,7 @@ golang.org/x/sys/windows
 golang.org/x/text/transform
 golang.org/x/text/unicode/norm
 golang.org/x/text/width
-# golang.org/x/tools v0.0.0-20190909030654-5b82db07426d => github.com/golangci/tools v0.0.0-20190910062050-3540c026601b
+# golang.org/x/tools v0.0.0-20190910044552-dd2b5c81c578 => github.com/golangci/tools v0.0.0-20190910062050-3540c026601b
```

Comment:
run `make vendor` to remove it please

Reply:
`make vendor` didn't work as it modified the `go.mod` and this file to what it was. Had to undo it manually. Fixed now.